### PR TITLE
Expanded README

### PR DESCRIPTION
**Pull request by amouat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,88 @@ This image can be used with `ko build`, `docker`, etc, but
...
+...
+$ docker run c-distroless
+Hello Distroless!
+$ docker images c-distroless
+REPOSITORY     TAG       IMAGE ID       CREATED              SIZE
+c-distroless   latest    f3648380711c   About a minute ago   2.88MB
+```
+
+For Go programs, we recommend using [ko](https://github.com/google/ko) and setting
+the `defaultBaseImage` to `distroless.dev/static`.
```
Comment (imjasonh) [inline README.md:67]: This will soon be the default: https://github.com/google/ko/issues/722 😎

File: README.md [hunk 2]
```diff
@@ -11,3 +11,88 @@ This image can be used with `ko build`, `docker`, etc, but
...
+# syntax=docker/dockerfile:1.4
+FROM gcc:latest as build
+
+COPY <<EOF /hello.c
+#include <stdio.h>
+int main() { printf("Hello Distroless!"); }
+EOF
+RUN cc -static /hello.c -o /hello
+
+FROM distroless.dev/static:latest
```
Comment (imjasonh) [inline README.md:53]: If we recommend distroless.dev, I'd like to opensource the redirector, so folks can be a little more sure we're not pulling any underhanded tricks.

File: README.md [hunk 3]
```diff
@@ -11,3 +11,88 @@ This image can be used with `ko build`, `docker`, etc, but
 is only suitable for running static binaries.
 
 This image is also regenerated nightly.
+
+## Usage
+
+Here's an example of building a Rust static binary with Docker tooling:
+
+```
+$ cat Dockerfile
```
[the remark this answers is not in the document]
Comment (amouat) [reply README.md:20]: Yup, I have basically that in another PR :) I didn't have it under examples though, I like that idea.

File: README.md (hunk 1 above)
Comment (amouat) [reply README.md:67]: Nice!

File: README.md (hunk 2 above)
Comment (amouat) [reply README.md:53]: Good point. I would like to recommend distroless.dev, so please go ahead AFAIC.

**Pull request merged**